code/js/i18n/cs.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@metadata": {
+    "author": "Roman",
+    "locale": "cs"
+  },
+  "rcalc-title": "Kalkulačka",
+  "rcalc-h1": "Romanova Kalkulačka",
+
+  "rcalc-dropdownButtonColor": "Barva",
```

Comment:
Format of keys will following. 
Words will split with dots --> rcalc.dropdown.button.color
Translations key will start with page where is it. Currently we have just one page so lets name it as 'calculator' --> calculator.dropdown.button.color ... for future is fine this format because of finding of translations.

Reply:
the format of translation's keys has been modified to calculator.dropdown.button.color

Reply:
Nice.

Reply:
sorry, but I get that... what do you mean here?

Reply:
caclculator

Reply:
you write above: Translations key will start with page where is it. I am still confused, do you mean the correct format should be index.dropdown.button.language?

Reply:
Correction: type --> typo (mistake in word spelling) ... so you should repair line 19 ... caclculator is no word ... I think

Reply:
misspelling again... corrected